### ISSUE TITLE
Request for parameter_weights

### ISSUE THREAD

**Issue by yubing0614** [mllam/neural-lam#660]
I am interested in evaluating the inference accuracy of several weather forecasting models and making a comparison analysis. However, I failed to find the corresponding parameter_weights in this model, which was crucial to the evaluation progress. Since I only wanted to run the inference code using your pre-trained model checkpoints, I thought I may not need to run the create_parameter_weights using the true training dataset because it was actually a huge work. So could you provide the parameter_weights of the training dataset for me to run the inference?

**Comment by yubing0614** [comment]
@claude @joeloskarsson have a look at this issue
